softbinding-algorithm-list.json

```
@@ -251,5 +251,20 @@
             "contact": "info.bornid@digicaps.com",
             "informationalUrl": "https://docs.bornid.io"
         }
+    },
+    {
+        "identifier": 18,
+        "alg": "com.sonicorigin.watermark.1",
+        "type": "watermark",
+        "decodedMediaTypes": [
+            "audio",
+            "video"
+        ],
+        "entryMetadata": {
+            "description": "SonicOrigin Watermarking Engine",
+            "dateEntered": "2025-06-16T17:00:00.000Z",
```

Could you update the date to the actual day the entry was sent on:

```suggestion
            "dateEntered": "2025-08-13T18:25:00.000Z",
```

---

@arosenMS confirms the change is okay to proceed with.